movie/src/components/header/AddMovie/AddMovie.tsx

```diff
@@ -0,0 +1,12 @@
+import { Button } from "@material-ui/core";
+import './AddMovie.scss';
+
+const AddMovie: React.FC = () => {
```

Comment:
You should name component clearly for each member of team. When someone will need to find a button is it a name which help to find a button that adding a new movie?
Try 'AddMovieButton' for example.

Reply:
renamed